src/crypto.js

```diff
@@ -40,12 +40,10 @@ exports.getRandomValues = function (buffer) {
     view = new Uint8Array(buffer),
     i;
 
-  if (offset + size > buf.length) {
-    throw new Error("Insufficient Randomness Allocated.");
-  }
-
+  // Hack: just wrap around.
```

Comment:
Yup, fully agreed.  But we're not generating enough randomness sometimes, and failing isn't helpful.  So, an alternative here would be to adjust this to be a promise, and let  tinvoke